### DISCUSSION TITLE
Language change Button

### DISCUSSION THREAD

**Discussion by Ansarmughal99** [florisboard/florisboard#2586] [Q&A]
I am unable to change language if we add Multiple  Language. There should be a button just like in the gboard which helps to switch the multiple languages. ![IMG_20240922_183441.jpg](https://github.com/user-attachments/assets/c5a25af7-cb06-472a-8b5b-6fe8be6fde0e)

**Comment by tsiflimagas** [comment]
I just tried it and it seems there is some issue with the Utility key action setting application. The default is dynamic selection between switch to emojis/switch languages. So, normally after you add a second language, it should automatically show the switch language icon, but it doesn't. And if you go and switch the setting repetitively to any option, most of the time it doesn't apply, or delays to do so. I'll open an issue for it.
